proto/config/devconfig.proto

```diff
@@ -170,6 +170,9 @@ message EdgeDevConfig {
 message ConfigRequest {
   string configHash = 1;
   bytes  integrity_token = 2; // value provided by controller during remote attestation
+  // Value of the field `seq` from the last received and successfully processed
+  // `CompoundEdgeDevConfig`.
+  uint64 ack_seq = 3;
```

Comment:
ConfigRequest message goes with the ConfigResponse message.

If you need to POST a message with additional fields and/or semantics then it would make sense to define a new CompoundEdgeDevConfigRequest. For instance, an integrity_token probably doesn't make any sense since there isn't necessarily an end-to-end remote attestation in the LOC case. So the semantics will be more clear with a new request message.